mypy/test/testconfigparser.py

```diff
@@ -0,0 +1,168 @@
+import os
+import tempfile
+from unittest import TestCase, main
+
+from mypy.config_parser import parse_config_file
+from mypy.options import Options
+
+
+class TestConfigParser(TestCase):
+    def test_parse_config_file_with_single_file(self) -> None:
+        """A single file should be correctly parsed."""
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            config_path = os.path.join(tmpdirname, "test_config.ini")
+
+            with open(config_path, "w") as f:
+                f.write(
+                    """
+                    [mypy]
+                    files = file1.py
+                    """
+                )
```

Comment:
Same here. pathlib has `Path.write_text()` which is a more straightforward API. It'll also remove the need to stick `f`s into places where you don't know how to name variables.